### ISSUE TITLE
Consul not reloaded when TLS files change

### ISSUE THREAD

**Issue by danielkucera** [ansible-collections/ansible-consul#256]
Consul is not reloaded when TLS files change.

**Comment by soloradish** [comment]
Agents only reload when config file changes. 

https://github.com/brianshumate/ansible-consul/blob/4507528909cfd6a30295f952ce721220514bb605/tasks/config.yml#L26

https://stackoverflow.com/a/35604895/2366058

**Comment by danielkucera** [comment]
yes @soloradish , that is what this issue is about. the same `notify` probably needs to be defined in: https://github.com/brianshumate/ansible-consul/blob/master/tasks/tls.yml
